test/createEpicMiddleware-spec.js

```diff
@@ -20,13 +22,29 @@ describe('createEpicMiddleware', () => {
     const mockMiddleware = store => next => action => {
       expect(epic.calledOnce).to.equal(true);
       expect(epic.firstCall.args[0]).to.be.instanceOf(ActionsObservable);
-      expect(epic.firstCall.args[1]).to.equal(store);
+      expect(epic.firstCall.args[1].getState).to.equal(store.getState);
       done();
     };
     const store = createStore(reducer, applyMiddleware(epicMiddleware, mockMiddleware));
     store.dispatch({ type: 'FIRST_ACTION_TO_TRIGGER_MIDDLEWARE' });
   });
 
+  it('should warn about improper use of dispatch function', () => {
+    sinon.spy(console, 'warn');
```

Comment:
Missing a matching restore of the original at the end of the test, `console.warn.restore()`